sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -611,7 +620,15 @@ private BasicDocumentRevision createDocument(SQLDatabase db,String docId, final
             // * normal insert logic for a new document
 
             InsertRevisionOptions options = new InsertRevisionOptions();
-            BasicDocumentRevision potentialParent = this.getDocumentInQueue(db, docId, null);
+            BasicDocumentRevision potentialParent = null;
```

Comment:
Remove the unused `documentCreated` variable above, and indent all of this left one level to be inline with the above test.